firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarMiddleware.swift

```diff
@@ -85,6 +85,13 @@ class ToolbarMiddleware: FeatureFlaggable {
         a11yLabel: .TabToolbarSearchAccessibilityLabel,
         a11yId: AccessibilityIdentifiers.Toolbar.searchButton)
 
+    lazy var dataClearanceAction = ToolbarActionState(
+        actionType: .dataClearance,
+        iconName: StandardImageIdentifiers.Large.dataClearance,
+        isEnabled: true,
+        a11yLabel: .TabToolbarDataClearanceAccessibilityLabel,
+        a11yId: AccessibilityIdentifiers.Toolbar.dataClearanceButton)
```

Comment:
I think we should use the existing a11yId (`AccessibilityIdentifiers.Toolbar.fireButton`) here instead of creating a new one as this could cause existing tests to fail.